reactor-netty-http/src/test/java/reactor/netty/http/server/logging/AccessLogTest.java

```diff
@@ -60,17 +56,15 @@ void setUp() {
 	}
 
 	@AfterEach
-	void tearDown() {
-		if (disposableServer != null) {
-			disposableServer.disposeNow();
-		}
+	@Override
```

Comment:
the method can be renamed, alternatively to overriding and calling `super`

Reply:
fixed